### ISSUE TITLE
Any way to see what Pokemon your opponent has?

### ISSUE THREAD

**Issue by rokaHakor** [dramamine/leftovers-again#15]
state.opponent.reserve only contains Pokemon that you have seen. I want to be able to get the Pokemon my opponent has based on the teamviewer.

**Comment by dramamine** [comment]
Yep this should be possible, need to read the 'poke' action from the server, ex this comes at the start of the match

```
|poke|p1|Wynaut, L5, F|item
|poke|p2|Trapinch, L5, F|item
|teampreview
```
You'd need to add a handler for this message in battle.js

Not working on this now but if anyone has a PR I'll take a look.
